genai-perf/genai_perf/telemetry_data/telemetry_data_collector.py

```diff
@@ -46,6 +46,16 @@ def __init__(
         self._stop_event = Event()
         self._thread: Optional[Thread] = None
 
+    def check_url_reachability(self) -> bool:
```

Comment:
```suggestion
    def is_url_reachable(self) -> bool:
```

Reply:
With boolean-returning functions, naming like this is helpful to make it clear that the function returns a boolean and what it is.